services/explorer/src/components/Header/Header.effects.ts

```diff
@@ -55,7 +55,7 @@ export const useHeaderEffects = () => {
       number: searchBlockNumber,
       hash: searchHash,
     }),
-    pause: !searchBlockNumber && !searchHash,
+    // pause: !searchBlockNumber && !searchHash,
```

Comment:
Should these be commented?
If we don't need them let's just delete it

Reply:
Removed